src/sections/about.html

```diff
@@ -0,0 +1,5 @@
+<section class="flex flex-col items-center justify-center px-10 py-20 my-20 lg:px-36">
+    <span class="text-3xl font-medium md:text-4xl lg:text-5xl">
```

Comment:
This content is too large for a `<span>` tag. I suggest using a `<p>` tag for this. However, you can use a `<span>` if the content is just a single sentence.